packages/fxa-auth-server/lib/oauth/token.js

```diff
@@ -35,7 +36,20 @@ exports.getTokenId = async function getTokenId(accessToken) {
  * Verify an accessToken.
  */
 exports.verify = async function verify(accessToken) {
-  // JWT access tokens are still database backed, continue
+  // This block is a special case for Sync atm. We don't store these in the db.
+  // Eventually we won't store any JWT tokens.
+  if (!validators.jwt.validate(accessToken).error) {
+    const t = await JWTAccessToken.verify(accessToken);
+    const scope = ScopeSet.fromString(t.scope);
+    if (scope.contains(OAUTH_SCOPE_OLD_SYNC)) {
+      return {
+        user: t.sub, // TODO: need to consider PPID?
```

Comment:
You shouldn't need to consider PPID for sync, I don't think.

If you didn't want to special-case this for sync, you could special-case it for "JWTs expiring in less than 6 hours don't need to be looked up in the db", by a parallel logic to the one you use when deciding whether to write them into the db.  (That will avoid hitting the db for any JWT that is "expiring soon", but that seems OK to me).